tests_metricflow/integration/query_output/test_query_output.py

```diff
@@ -93,3 +93,80 @@ def test_derived_metric_alias(  # noqa: D103
         snapshot_str=query_result.result_df.text_format(),
         sql_engine=sql_client.sql_engine_type,
     )
+
+
+@pytest.mark.sql_engine_snapshot
+@pytest.mark.duckdb_only
+def test_scd_with_coarser_grain(  # noqa: D103
```

Comment:
It might be worth grouping by the listing as well so we can see when the same one is listed multiple times, and then have the docstring explain what's expected. Up to you if you want to do this.